user-licence-lifecycle-manager/src/main/resources/META-INF/update-licence-table.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog-3.1.xsd">
+    <changeSet author="leantios" id="add-updated-at-column">
+        <addColumn tableName="LICENCE">
+            <column name="UPDATED_AT" type="TIMESTAMP">
+                <constraints nullable="true"/>
```

Comment:
make it not-nullable please and add:
```xml
<update tableName="LICENCE">
  <column name="UPDATED_AT" valueComputed="get_created_at_here()"/>
  <where>updated_at=CHECK_NULL_CORRECTLY</where>
</update>
```
https://docs.liquibase.com/change-types/update.html#xml_example